src/dummyDB/resourcesData.js

```diff
@@ -0,0 +1,34 @@
+
+const resourcesData = [
+  {
+    resource:{
+      id: 12345,
+      parent: 54321,
```

Comment:
change this seed data to have an object for parent:

```js
parent: {
  id: 54321,
  title: 'Some Title'
}
```

Reply:
updated seed data with this new model